bench/plutus-scripts-bench/src/Cardano/Benchmarking/PlutusScripts.hs

```diff
@@ -10,39 +10,40 @@ module Cardano.Benchmarking.PlutusScripts
     , findPlutusScript
     , getAllScripts
     , listPlutusScripts
+    , asAnyLang
+    , normalizeModuleName
     ) where
 
 import           Prelude
 
 import           Data.ByteString.Lazy as LBS (ByteString)
+import           Data.Text(split, pack)
+import           Data.Maybe(listToMaybe)
+import           System.FilePath(takeBaseName)
 
 import           Cardano.Api
 
 import qualified Cardano.Benchmarking.PlutusScripts.CustomCall as CustomCall
 import qualified Cardano.Benchmarking.PlutusScripts.EcdsaSecp256k1Loop as ECDSA
 import qualified Cardano.Benchmarking.PlutusScripts.Loop as Loop
 import qualified Cardano.Benchmarking.PlutusScripts.SchnorrSecp256k1Loop as Schnorr
+import           Cardano.Benchmarking.ScriptAPI
 
 
-getAllScripts ::
-     [(String, ScriptInAnyLang)]
+getAllScripts :: [BenchScript]
 getAllScripts =
-  [ (normalizeModuleName CustomCall.scriptName, asAnyLang CustomCall.scriptSerialized)
-  , (normalizeModuleName ECDSA.scriptName     , asAnyLang ECDSA.scriptSerialized)
-  , (normalizeModuleName Loop.scriptName      , asAnyLang Loop.scriptSerialized)
-  , (normalizeModuleName Schnorr.scriptName   , asAnyLang Schnorr.scriptSerialized)
-  ]
+  [ CustomCall.script, ECDSA.script, Loop.script, Schnorr.script ]
```

Comment:
This will have quicker to perceive diffs, if we break this line to list-entry-per-line format:
```suggestion
  [ CustomCall.script
  , ECDSA.script
  , Loop.script
  , Schnorr.script
  ]
```